examples/pytorch/graphsage/model.py

```diff
@@ -0,0 +1,150 @@
+import torch
+import torch.nn as nn
+import dgl
+from dgl import DGLGraph
+from dgl.nn.pytorch.conv import SAGEConv
+import tqdm
+
+from dgl.multiprocessing import shared_tensor
+from dgl.utils import pin_memory_inplace, unpin_memory_inplace
+import torch.distributed as dist
+
+class GraphSAGE(nn.Module):
+    def __init__(self,
+                 in_feats,
+                 n_hidden,
+                 n_classes,
+                 n_layers,
+                 activation,
+                 dropout,
+                 aggregator_type):
+        super().__init__()
+        self.layers = nn.ModuleList()
+        self.dropout = nn.Dropout(dropout)
+        self.activation = activation
+
+        # input layer
+        self.layers.append(SAGEConv(in_feats, n_hidden, aggregator_type))
+        # hidden layers
+        for i in range(n_layers):
+            self.layers.append(SAGEConv(n_hidden, n_hidden, aggregator_type))
+            # output layer
+        self.layers.append(SAGEConv(n_hidden, n_classes, aggregator_type)) # activation None
```

Comment:
The comment `# activation None` might be unnecessary.

Reply:
Okay. I will remove the comment